neptune/new/internal/backends/utils.py

```diff
@@ -99,6 +100,10 @@ def wrapper(*args, **kwargs):
 
             try:
                 return func(*args, **kwargs)
+            except requests.exceptions.InvalidHeader as e:
```

Comment:
that sounds like a perfect scenario for an e2e test, don't you think? :D

Reply:
ok, it turns out it's surprisingly hard to get into that branch after adding `.strip()` -- so I'm resolving that